### PR TITLE
 [plugin.video.embycon] 1.3.40

### PR DESCRIPTION
**Pull request by faush01**
### Description

Latest version of EmbyCon
 - New action menu for widgets
 - New Widget actions for skins
 - Many fixes
### Checklist:

- [ ] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [ ] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [ ] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.video.embycon/addon.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon  id="plugin.video.embycon" 
         name="EmbyCon"
-        version="1.3.24"
+        version="1.3.40"
         provider-name="Team B">
   <requires>
     <import addon="xbmc.python" version="2.24.0"/>
```
Comment (Lunatixz) [inline plugin.video.embycon/addon.xml]: Please change to 2.25.0

**Comment by faush01** [comment]
From memory this stops the addon being comparable with Kodi 16 though.

**Comment by Lunatixz** [comment]
Correct, however you are committing this PR to Krypton. The Jarvis branch is closed.

**Comment by faush01** [comment]
People on Kodi 16 are still using the add-on and update from the test repo I have set up, changing this would break that option for them. Alternatively I would need to create a new add-on that is Kodi 16 comparable as I can't have two add-ons in the same repo with the same id, I don't this that is manageable.

**Comment by Lunatixz** [comment]
I understand, however this commit was originally made to Krypton... The Jarvis branch is closed. I would suggest you create a new branch in your personal repo for "jarvis" builds and continue to update the kodi repository with Krypton updates. Jarvis users with your repository will receive updates, while their systems ignore Krypton updates via the Official repository.

**Comment by faush01** [comment]
That's the problem, maintaining both, keeping them updated. These is no reason for it, the addon works on both but now I need a seperate branch to support both.

**Comment by Lunatixz** [comment]
http://kodi.wiki/view/Submitting_Add-ons#Allowed_submissions

**Comment by faush01** [comment]
updated, I will work out what I am going to do with K16 some other time.

**Comment by learningit** [comment]
@faush01 I don't think that made a difference anyway. A Jarvis user would not see your add-on in the kodi.org repo because it is in the Krypton branch, regardless the xbmc.python being set for Jarvis.

**Comment by MartijnKaijser** [comment]
correct. What's in Krypton will only be in Krypton and up. V16 is dead to us so we don't want new addons for it. I'm not opposed to having a lower xbmc.python version in case they want to install it manually

**Comment by faush01** [comment]
Thanks for the clarification.
That is exactly what I wanted to do, just have it so users could install the addon manually if they wanted to or needed to for Kodi 16. I never intended to actually submit the addon to the old Kodi 16 addon branch.

**Comment by faush01** [comment]
So can we merge this change then?

**Comment by faush01** [comment]
Can someone please explain what I need to do here to have this merged.

I just need clear directions on what is holding up the process so I can fix it.

**Comment by Lunatixz** [comment]
@faush01 I believe you need a second review; larger programs such as your's requires more time. From what I can tell you are good to go after a second review :).

**Comment by Lunatixz** [comment]
@enen92 @Razzeee LGTM

**Pull request merged**